cloudconfig/containerinit/container_userdata.go

```diff
@@ -202,17 +202,19 @@ func PrepareNetworkConfigFromInterfaces(interfaces []network.InterfaceInfo) *Pre
 // might include per-interface networking config if both networkConfig
 // is not nil and its Interfaces field is not empty.
```

Comment:
Is this comment still valid? Looks like a change in behavior, I would have liked to see a change in comment too \o/